### PR TITLE
Fixes #111: Build on #113 also address `Unable to find file /etc/selinux/config` error

### LINKED ISSUE #111: [BUG][No module named 'selinux' during executing of task "Disable the selinux"]
**Describe the bug**
When task "Disable the selinux" runs on OS without installed SELinux, for example Debian, it is failed with error: "An exception occurred during task execution. The error was: ModuleNotFoundError: No module named 'selinux'". 

**To Reproduce**
Run playbook opensearch.yaml to install system on debian OS without installed selinux.
The playbook is failed with error "ModuleNotFoundError: No module named 'selinux'"

**Expected behavior**
Playbook ignore missing selinux

**Playbook Name**
opensearch.yml playbook - roles linux - opensearch - tasks main - task "Disable the selinux"
opensearch.yml playbook - roles linux - dashboards - tasks main - task "Disable the selinux"

**Host/Environment (please complete the following information):**
ansible [core 2.14.2]
python version = 3.9.2
jinja version = 3.1.2
Playbook Version: 2.3.0

### PR DESCRIPTION
**Pull request by heiderich**
### Description

This is build on top of #113 of @FactorT.

### Issues Resolved

#111

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

Note that this commit also contains two commits (c5fe1e2ea87b1642585752abea4aa1e3499ce879, cddafd6d00da23225d7f3e68a33e822647cb183c) of @FactorT from #113.

### CODE REVIEW

**Comment by prudhvigodithi** [comment]
Hey @heiderich thanks for the contribution, so once this PR is merged, we dont need https://github.com/opensearch-project/ansible-playbook/pull/113?

**Comment by heiderich** [comment]
@prudhvigodithi Yes, exactly. The two commits from #113 are included in this PR.

File: roles/linux/dashboards/tasks/main.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@
   selinux:
     state: disabled
   when: (ansible_distribution != "Ubuntu") and (ansible_distribution != "Amazon")
```
Comment (saravanan30erd) [inline roles/linux/dashboards/tasks/main.yml:10]: @heiderich Initially I added `when` condition to do this task only for OS distribution other than ubuntu and amazon linux where I faced different errors due to unavailable of this module in those distributions.

If this [condition](https://github.com/opensearch-project/ansible-playbook/pull/122/files#diff-26da4e2fc3ae1d719ad41b3e625de34dd3fbc83814e8a4c6875c55f4fb6f0e91R12) covers errors from `Ubuntu` and `amazon` distributions then we can remove `when` completely.

Comment (saravanan30erd) [reply roles/linux/dashboards/tasks/main.yml:10]: @heiderich could you please check the above comments?

**Comment by TheAlgo** [comment]
@ng-bsy I think there is an open comment by @saravanan30erd which is still open for reply from @heiderich  and hence the PR is still open.

**Comment by prudhvigodithi** [comment]
Thanks @TheAlgo and thanks @ng-bsy for your patience, its true we are holding based on pending conversation, @saravanan30erd are we ok to move forward with this PR based on above [comment](https://github.com/opensearch-project/ansible-playbook/pull/122#issuecomment-1814675779)?

**Comment by saravanan30erd** [comment]
@prudhvigodithi @peterzhuamazon The concern with this PR is, the way its managed to ignore the errors are limited and not scalable. For example, overall issues are occurring for debian based on this source issue https://github.com/opensearch-project/ansible-playbook/issues/111 but for different versions of python the errors are also different for same debian distribution. As you can see, here https://github.com/opensearch-project/ansible-playbook/issues/111#issue-1575639716 its different error output and here its another one https://github.com/opensearch-project/ansible-playbook/issues/111#issuecomment-1474984451 thats the reason here https://github.com/opensearch-project/ansible-playbook/pull/122/commits/c4b21560b69a57af97261468e36857e7906afe4c#diff-26da4e2fc3ae1d719ad41b3e625de34dd3fbc83814e8a4c6875c55f4fb6f0e91R12 new error output is added. In this case, we might get new error output for new versions and we need to keep on adding those new changes in the `search` thats why I am pointing out its not scalable.
I can see both issues were for debian, so we can ignore completely that distribution for this step so we don't have to cover different error outputs to ignore. Raised the pr for this https://github.com/opensearch-project/ansible-playbook/pull/152 please review it

**Comment by dblock** [comment]
Thank you for the contribution @heiderich. I think the consensus is not to merge this PR because of the reasons above, @prudhvigodithi should we close it?

**Comment by peterzhuamazon** [comment]
Close this as it has been handled in #152.

Thanks.

**Pull request closed without merging**